client/src/ActionType.js

```diff
@@ -3,4 +3,5 @@ export const DARK_THEME_INACTIVE = 'DARK_THEME_INACTIVE';
 export const SIGNED_IN_SUCCEEDED = 'SIGNED_IN_SUCCEEDED';
 export const RESET_SIGNED_IN_SUCCEEDED = 'RESET_SIGNED_IN_SUCCEEDED';
 export const USER_INFO = 'USER_INFO';
+export const MILITARY_VERSION = 'MILITARY_VERSION';
```

Comment:
You can add SPORTS_VERSION variable too.

Reply:
sure I will add it.

Reply:
added